src/pages/vaults/VaultPositionsTable.tsx

```diff
@@ -92,17 +92,17 @@ export const VaultPositionsTable = ({ className }: { className?: string }) => {
                   >
                     {marketId === USDC_MARKET_HARDCODED ? usdcLabel : asset?.name}
                     <div tw="row gap-0.25">
-                      <$OutputSigned
-                        value={
-                          (currentLeverageMultiple ?? 0) === 0
-                            ? stringGetter({ key: STRING_KEYS.NONE })
-                            : (currentLeverageMultiple ?? 0) < 0
+                      {(currentLeverageMultiple ?? 0) !== 0 && (
```

Comment:
is this different from 
`!!currentLeverageMultiple`? since what we're doing here is having nully values convert to 0, which then evaluate to false?

Reply:
true that works too, I will do that